modules/installation-osp-enabling-swift.adoc

```diff
@@ -0,0 +1,40 @@
+//Module included in the following assemblies:
+//
+// * installing/installing_openstack/installing-openstack-installer.adoc
+// * installing/installing_openstack/installing-openstack-installer-custom.adoc
+
+[id="installation-osp-enabling-swift_{context}"]
+= Enabling Swift on OpenStack
+
+{product-title} on OpenStack uses https://docs.openstack.org/security-guide/object-storage.html[OpenStack Object Storage (Swift)] to store and serve user configuration files.
+
+//[Ignition](https://coreos.com/ignition/docs/latest/what-is-ignition.html)
+
+Swift is operated by a user account with the `swiftoperator` role and `temp-url` support.
+
+.Prerequisites
+
+* An OpenStack administrator account on the target environment
+
+.Procedure
+
+To enable Swift on OpenStack:
+
+. In the OpenStack CLI, add the `swiftoperator` role to the account that will access Swift.
++
+[source, sh]
+----
+$ openstack role add --user <user> --project <project> swiftoperator
+----
+. Set a temporary URL property for the account.
++
+[source, sh]
+----
+$ openstack object store account set --property Temp-URL-Key=superkey
+----
+
+Your OpenStack deployment can now use Swift to store and serve files.
+
+[NOTE]
+//When?
```

Comment:
Yeah, the security groups have nothing to do with Swift, sorry if the upstream docs led you to think otherwise.

You can just delete this note. The values here are incorrect (they should match the quotas in the table of the main IPI/customisation doc.

If you want, you can document the commands on setting these quotas there, but in that case we should probably have commands that set up everything, not just values related to the security groups.